python/tvm/script/context_maintainer.py

```diff
@@ -74,6 +76,10 @@ def example_func(a: T.handle, b: T.handle, c: T.handle) -> None:
     """List[Buffer]: list of T.alloc_buffer statements in the block signature"""
     match_buffers: List[MatchBufferRegion] = []
     """List[MatchBufferRegion]: list of T.match_buffer statements in the block signature"""
+    axes: List[Axis] = []
+    """List[Axis]: list of sparse axis created in the block signature."""
+    match_sparse_buffers: List[MatchSparseBuffer]
+     
```

Comment:
Should have a comment here for `match_sparse_buffers` :eyes:?

Reply:
Sorry I deleted it by mistake.